File: 0001-two-sum/0001-two-sum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        result = []
        for i in range(0,len(nums)-1):
            for j in range(i+1,len(nums)):
                if nums[i] + nums[j] == target:
                    result.extend([i,j])
                    return result

File: 0001-two-sum/0001-two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        for (int i = 0; i < nums.length - 1; i++) {
            if (nums[i] + nums[i + 1] == target) return new int[]{i, i + 1};
        }

        for (int i = 0; i < nums.length; i++) {
            int temp = target - nums[i];

            for (int j = i + 1; j < nums.length; j++) {
                if (temp == nums[j]) return new int[]{i, j};
            }
        }

        return new int[]{-1, -1};
    }
}
